package.json

```diff
@@ -1,19 +1,23 @@
 {
-  "name": "branch-name",
+  "name": "branch-name-checker",
   "private": true,
   "main": "index.js",
   "scripts": {
     "start": "node ./index.js",
+    "build": "tsc",
     "test": "jest --coverage"
   },
   "dependencies": {
-    "actions-toolkit": "^4.0.0",
-    "dotenv": "^8.2.0"
+    "actions-toolkit": "6.0.1"
```

Comment:
I have increased the versions of the dependencies in this PR since I have upgraded the Node version. If we merge the PR https://github.com/AlbertHernandez/branch-name-action/pull/27 before, I think we can keep the package this way.

Reply:
If we need to change the version of the library due to node version, most probably this change should be done in the other PR and not here

Reply:
Changed in the other PR the dependency versions.

Reply:
Nice, then we will wait until we have the other PR merged to continue with this one